src/components/Main/Main.tsx

```diff
@@ -211,29 +212,36 @@ const Main: React.FC<IMainProps> = ({ handleBackgroundColor }) => {
       <Card className={classes["timer_container"]}>
         <div className={classes["timer_container__change-timertype_buttons"]}>
           <Button
-            className={`${
-              state.timerType === POMODORO
-                ? classes["btn--active"]
-                : classes["btn"]
-            }`}
+            className={classNames({
+              [classes["btn--active"]]: state.timerType === POMODORO,
+              [classes["btn"]]: state.timerType !== POMODORO,
```

Comment:
esto está raro, por qué son clases opuestas? el modificador "active" debería ser sólo extensible a btn, pero acá veo que ambos son mutuamente excluyentes, lo que no debería ocurrir.

Reply:
Lo correcto sería algo así:
```suggestion
              classes.btn, // mantiene la estructura basica del botón
              [classes["btn--active"]]: state.timerType === POMODORO, // extiende o modifica el diseño base.
```